.github/workflows/compfuzzci_no_forks.yaml

```diff
@@ -0,0 +1,91 @@
+name: CompFuzzCI
```

Comment:
Could you add a comment to the top of this file that explains what it does?

Reply:
Good idea. I'll get that done. Thanks!